autoload/phpactor/project/projectRepository.vim

```diff
@@ -0,0 +1,42 @@
+function! phpactor#project#projectRepository#create() abort
```

Comment:
What about renaming the file in order to have a function like `phpactor#project#repository#create()`.
IMO repeating the `project` is unnecessary and disgracious :)